civihr_employee_portal/civihr_employee_portal.drush.inc

```diff
@@ -100,6 +100,14 @@ function drush_civihr_employee_portal_refresh_node_export_files() {
       }
 
       node_export_save($importNode);
+
+      // Need to manually update webform_civicrm_form record
+      if ($importNode->type === 'webform' && !$importNode->is_new) {
```

Comment:
This change needs to happen only when the Imported node is not new?

Reply:
I've tested it and when it is new it gets saved as expected. I think it might be a problem with webform_civicrm module where the same code is not run on update.